src/svgcontext.ts

```diff
@@ -386,6 +384,12 @@ export class SVGContext extends RenderContext {
     return this;
   }
 
+  pointerRect(x: number, y: number, width: number, height: number): this {
+    const attributes = { opacity: '0', 'pointer-events': 'auto' };
+    this.rect(x, y, width, height, attributes);
+    return this;
+  }
+
```

Comment:
So this solution draws an invisible rectangle on top of the glyph to capture events? Does that mean we are adding an extra shape for every note? Does this invisible rect cover the stems and flags and other parts, in addition to the note head?

I did verify that v4 mouseovers worked, while v5 glyphs have really large boxes that breaks the mouseovers (i.e., you can hover over empty space far above the glyph and it will still trigger the hover highlight). We might need to investigate to see if there is a better way.

Reply:
Yes, it covers the box returned by StaveNote.getBoundingBox(). This is noteheads, stem and flag. No accidentals/ornaments/dots...

This is the only idea that I could think of to resolve the issue in  https://github.com/vexflow/vexflow/issues/155#issuecomment-1803319476

Reply:
Yeah that's a tough one to solve. Ideally we only add these invisible rects if the developer wants to turn on interactivity. If a score is not interactive, we shouldn't need to add all these extra unused elements. 

Let me think about this for a bit. I can merge this weekend if I can't think of any better solution.